drivers/auscom/cpl_arrays_setup.F90

```diff
@@ -62,9 +62,11 @@ module cpl_arrays_setup
 !(15) ice form waterflux				ioform
 !(16) land ice waterflux                iolicefw
 !(17) land ice heatflux                 iolicefh
+!(18) 10m wind speeds                                   iownd10
 !
 !
-! 18 in, 18 out => thus we set jpfldout=18, jpfldin=18 (in cpl_parameters)! 
+
+! 18 in, 18 out => thus we set jpfldout=18, jpfldin=18 (in cpl_parameters)! RASF Check this statement.
```

Comment:
@russfiedler what needs to be checked with this statement?

Reply:
The statement is inconsistent with the information above it. In the old version there were 17 in and 17 out listed above it not 18. I think @nichannah 's changes have made all this obsolete anyway as jpfldin/out no longer exist.